packages/verification-vue-component/src/components/Verification/item/cards/SigningCard.vue

```diff
@@ -82,19 +82,37 @@
                         </template>
                     </DataPanel>
                 </div>
+                <div v-if="retractEvents.length > 0 && retractEvents[0].on_blockchain" class="section retraction-hash">
```

Comment:
I guess "revoked" is not used anymore if it's a signing process, so can we remove the revoke logic? Or should we combine revoke events/claims into retracted events? From now on it shouldn't be possible anymore to revoke signed documents, right?

Reply:
Yes revoking is not possible anymore for signed files but we still have revoked files and show them as revoked

Reply:
Yes, the question is do we still want to display them as revoked or would it be ok to just display them as retracted?

Reply:
Yes we already discussed this, we keep them as revoked

Reply:
Ok